org/wildfly/undertow/application-security-domain/capability.adoc

```diff
@@ -0,0 +1,26 @@
+NAME: org.wildfly.undertow.application-security-domain
+
+DESCRIPTION: Provides the mapping from a security domain as specified in a web application
+              to an HttpAuthenticationFactory plus additional policy information.
+
+REGISTERED BY:
+
+  NAME: WildFly
+  URL:  https://github.com/wildfly/wildfly
+
+DYNAMIC: true
+
+PRIVATE: true
+
+SUPPORTS RUNTIME ONLY: true
+
+SERVICE PROVIDED:
+
+   CLASS: java.util.function.BiFunction<DeploymentInfo, Function<String, RunAsIdentityMetaData>, Registration>
```

Comment:
The only use I saw in clustering was clustering/web/undertow/src/main/java/org/wildfly/clustering/web/undertow/sso/elytron/DistributableSingleSignOnManagerBuilder.java:        

```
ServiceName securityDomainServiceName = support.getCapabilityServiceName(ApplicationSecurityDomainDefinition.APPLICATION_SECURITY_DOMAIN_CAPABILITY, securityDomainName);

Builder<SSOManagerFactory<ElytronAuthentication, String, Map.Entry<String, URI>, Batch>> factoryBuilder = PROVIDER.<ElytronAuthentication, String, Map.Entry<String, URI>>getBuilder(securityDomainName).configure(support);
        Builder<SessionIdGenerator> generatorBuilder = new SimpleBuilder<>(securityDomainServiceName.append("generator"), generator);
```

But all that's doing is using the string constant to create it's own service name with "generator" appended. It doesn't actually use the capability.